src/styles/normalize.scss

```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
This could be imported from https://www.npmjs.com/package/normalize.css to get notified of updates.